### PR TITLE
port could be configurable

### PR DESCRIPTION
**Pull request by jenskuespert**
Hi,

I needed to use port different from 443, so I added a new optional parameter.

Regards,
-- 
    - Jens -

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ optional arguments:
 API Options:
   -H HOSTNAME, --hostname HOSTNAME
                         OPNsense hostname or ip address
+  -H PORT, --port PORT  OPNsense https-api port
```
Comment (nbuchwitz) [inline README.md]: Can you please fix the first parameter? Should be `-p`

**Review by nbuchwitz — CHANGES_REQUESTED** [review]
Thanks for the PR. Can you please fix the typo in the doc. I'll merge asap

File: README.md (hunk 1 above)
Comment (jenskuespert) [reply README.md]: Thanks for the consideration. As far as I can see, you already fixed it - what should I do now?

Comment (nbuchwitz) [reply README.md]: Sorry for the confusion, I'll merge it.

**Pull request merged**